tests/test_models/test_losses/test_mse_loss.py

```diff
@@ -0,0 +1,25 @@
+import torch
+
+from mmhuman3d.models import build_loss
+
+
+def test_keypoint_mse__loss():
```

Comment:
`test_keypoint_mse__loss` has a double underscore `__`.

Reply:
Resolve in https://github.com/open-mmlab/mmhuman3d/pull/102/commits/05502deaf6738d59f9b99664233deb74eaeca649